File: src/test/java/com/brian/StudentBeanTest.java
package com.brian;

import com.brian.student.StudentBean;
import org.junit.Test;
import org.junit.jupiter.api.Assertions;

public class StudentBeanTest {

    @Test
    public void highTest() {
        StudentBean studentBean = new StudentBean("Peter", 50, 80);
        Assertions.assertEquals(80, studentBean.highest());
    }

    @Test
    public void averageTest() {
        StudentBean studentBean = new StudentBean("Peter", 50, 80);
        Assertions.assertEquals(65, studentBean.average());
    }

}

File: src/test/kotlin/com/student/StudentBeanTest.kt
package com.student

import com.brian.Person2
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentBeanTest {
    @Test
    fun Person2Test() {
        var bmi = Person2(weight = 70f, high = 1.8f);
        Assertions.assertEquals(70f / (1.8f * 1.8f), bmi.bmi())
    }
}